add current limiting to shooter angle subsystem

josieammer opened the pull request.

josieammer committed:
add current limiting to shooter angle subsystem

mmcadamsms commented on ted/src/main/java/frc/robot/subsystems/ShooterAngleSubsystem.java:
```diff
@@ -186,6 +186,11 @@ private void configureAngleMotors() {
     TalonFXConfiguration angleConfigs = new TalonFXConfiguration();
     angleConfigs.MotorOutput.NeutralMode = NeutralModeValue.Brake;
     angleConfigs.MotorOutput.Inverted = Constants.angleLeftTalonShooterMotorDefaultDirection;
+    angleConfigs.CurrentLimits.StatorCurrentLimit = 100;
```
For shooter outfeed I put similar items up in Constants.java

mmcadamsms replied:
I just pushed a change as an example what we could do.

josieammer replied:
Looks good.

mmcadamsms committed:
Update to introduce hardware constants file.

Josie - how about this?  I think this will form a good place for me to put other hardware related stuff, for instance the new CAN related constants that will be needed.

josieammer committed:
Merge branch 'main' into ShooterAngleCurrentLimits
